DEVELOPING.md

```diff
@@ -127,6 +127,31 @@ For deploying and testing local AMP builds on [HEROKU](https://www.heroku.com/)
 
 Meantime, you can also use our automatic build on Heroku [link](http://amphtml-nightly.herokuapp.com/), which is normally built with latest head on master branch (please allow delay). The first time load is normally slow due to Heroku's free account throttling policy.
 
+To be able to test ads and third party frame make sure to `thirdPartyFrameHost`
```

Comment:
This is not what I was hoping we'd do.

Couldn't we automatically generate the right config for localDev and heroku and pre-prepend it using the `AMP_CONFIG` mechanism?